packages/component-base/test/virtualizer-unlimited-size.test.js

```diff
@@ -178,7 +178,10 @@ describe('unlimited size', () => {
   it('should have a last visible index at start', () => {
     const item = elementsContainer.querySelector(`#item-${virtualizer.lastVisibleIndex}`);
     const itemRect = item.getBoundingClientRect();
-    expect(scrollTarget.getBoundingClientRect().bottom).to.be.within(itemRect.top, itemRect.bottom);
+    expect(scrollTarget.getBoundingClientRect().bottom).to.be.within(
+      Math.round(itemRect.top),
+      Math.round(itemRect.bottom)
+    );
```

Comment:
Not related to the change, but these tests have started failing (on master) due to some sub-pixel rendering change.